src/contexts/Drive/Drive.context.tsx

```diff
@@ -77,6 +82,92 @@ export const DriveContextProvider: React.FC<DriveContextProviderProps> = ({ chil
     return driveUseCases.getFolderContent({ folderId });
   };
 
+  const getModifiedItems = async ({
```

Comment:
move this to driveFile.service and driveFolder.service to get each array of updated items, don't place this logic here

https://github.com/internxt/drive-mobile/blob/master/src/services/drive/file/driveFile.service.ts
https://github.com/internxt/drive-mobile/blob/master/src/services/drive/folder/driveFolder.service.ts